app/jobs/comments/send_email_notification_job.rb

```diff
@@ -4,7 +4,7 @@ class SendEmailNotificationJob < ApplicationJob
 
     def perform(comment_id)
       comment = Comment.find_by(id: comment_id)
-      NotifyMailer.new_reply_email(comment)&.deliver if comment
+      NotifyMailer.new_reply_email(comment).deliver_now if comment
```

Comment:
is there a reason why we were calling `deliver` instead of `deliver_now` being this already an async job? We call `deliver_now` in other similar jobs